crypt.go

```diff
@@ -550,6 +550,26 @@ func ActivateVolumeWithRecoveryKey(volumeName, sourceDevicePath string, keyReade
 	return activateWithRecoveryKey(volumeName, sourceDevicePath, keyReader, options.RecoveryKeyTries, RecoveryKeyUsageReasonRequested, activateOptions, options.KeyringPrefix)
 }
 
+// ActivateVolumeWithKey attempts to activate the LUKS encrypted volume at
+// sourceDevicePath and create a mapping with the name volumeName, using the
+// provided key. This makes use of systemd-cryptsetup.
+//
+// The ActivateOptions field of options can be used to specify additional
+// options to pass to systemd-cryptsetup. All other fields are ignored.
```

Comment:
I wonder whether it would be worth just passing a []string for the options instead?

Incidentally, I don't think anything is using these options anyway, and I wonder if there is actually a use-case for it. From looking at the options handled by systemd-cryptsetup, I'm not sure if any of them are actually useful to surface here.

Reply:
I think that for consistency with other Activate* calls we should stick to passing ActivateVolumeOptions, but that's just my non-binding opinion.

Reply:
I agree with @bboozzoo point at least until this is moved somewhere else. 

@chrisccoulson your comment about options not being useful was it specific to use the key directly or also for the other usages we have atm by the other ActivateVolume* functions?